src/services/index.ts

```diff
@@ -0,0 +1 @@
+export { default as FeeServices } from './fee.services'
```

Comment:
I (or another repo admin) may need to help you with publishing this folder.

1. add a package.json to this folder
2. add folder to lerna.json
3. add this folder to vite.config.js
4. `lerna bootstrap --hoist`
5. check if this folder got a package-lock.json file
6. commit everything
7. an admin will publish it after your PR is merged

Ref: https://github.com/bcgov/bcrs-shared-components#how-to-create-a-new-package-you-depend-on-eg-new-mixins-folder

Reply:
I followed the first 3 steps and on step 4 I got the message, "'lerna' is not recognized as an internal or external command,
operable program or batch file." when trying to run the command `lerna bootstrap --hoist`

Reply:
`npm install -global lerna`

Reply:
![image](https://github.com/bcgov/bcrs-shared-components/assets/122643780/ef0d5ed9-dced-445d-9226-8782f4fab4d3)
![image](https://github.com/bcgov/bcrs-shared-components/assets/122643780/8efb1143-b6fb-4ab6-b0eb-eaa1e2c94e0b)
I guess its already done by npm?

Reply:
Uninstall the latest Lerna and install this one instead: https://github.com/bcgov/bcrs-shared-components#4-install-dependencies